src/Components/SportsList.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState, useContext } from "react";
+import { SearchKeywordContext } from "./SearchKeywordContext";
+import "./SportsList.scss"
+
+function SportsList () {
+  const [data , setData] = useState([]);
+  const {searchKeyword} = useContext(SearchKeywordContext);
+  
+  useEffect(() => {
+    requestSportsList();
+  });
```

Comment:
This would run forever.
![image](https://user-images.githubusercontent.com/1394580/231106854-21319bb8-7ea7-49bc-9ccd-ccc4ba52d8f6.png)

Reply:
Super cool! Thanks.
Done.